src/python/create.py

```diff
@@ -532,4 +558,4 @@ def _get_relation(related_objs, cell_value, parsed_cell):
 
 def cellToBool(val):
     """Converts a cell value to a useable boolean"""
-    return True if str(val).lower() == "true" else False
+    return True if str(val).lower() != "no" else False
```

Comment:
Does it works in Python without the if/else?
```suggestion
    return str(val).lower() != "no"
```